src/ts/src/App.scss

```diff
@@ -1,38 +1,50 @@
-.App {
-  text-align: center;
+// Override some browser CSS defaults with better defaults.
+body {
+    margin: 0;
+    padding: 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
+// .App {
+//     text-align: center;
+// }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
+// .App-logo {
+//     height: 40vmin;
+//     pointer-events: none;
+// }
 
-.App-link {
-  color: #61dafb;
-}
+// @media (prefers-reduced-motion: no-preference) {
+//     .App-logo {
+//         animation: App-logo-spin infinite 20s linear;
+//     }
+// }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+// .App-header {
+//     background-color: #282c34;
+//     min-height: 100vh;
+//     display: flex;
+//     flex-direction: column;
+//     align-items: center;
+//     justify-content: center;
+//     font-size: calc(10px + 2vmin);
+//     color: white;
+// }
+
+// .App-link {
+//     color: #61dafb;
+// }
+
+// @keyframes App-logo-spin {
+//     from {
+//         transform: rotate(0deg);
+//     }
+//     to {
+//         transform: rotate(360deg);
+//     }
+// }
```

Comment:
this intentional?

Reply:
Yes, I meant to just delete it outright! Will fix that.